### DISCUSSION TITLE
Communication drops until I restart the WiFi access point

### DISCUSSION THREAD

**Discussion by jacoscar** [cmcgerty/Sofar2mqtt#72] [General]
Every few days the microcontroller stops communicating with the mqtt broker and the ‘RS485’ icon on the inverter lcd disappears.
I don't think this is the common problem that many have seen because it is not solved by resetting the inverter, but it’s solved by rebooting or restarting the access point (even without resetting the inverter).

It used to work fine until I started using an access point rather than the main router (having moved the inverter to the garage)
I’m on my second access point: the first one was this cheap one from amazon
WiFi Extender Booster,Wireless... https://www.amazon.co.uk/dp/B0C7BZQ5HG?ref=ppx_pop_mob_ap_share
I now have gone for a more know brand
TP-Link AC1200 Mesh Dual Band... https://www.amazon.co.uk/dp/B0929WHLPP?ref=ppx_pop_mob_ap_share

But I still have the same problem, and only with the sofar2mqtt Wemos D1; my other devices work fine
The Access Point has DHCP disabled, and DHCP is managed by the main router.
Any suggestions?

**Comment by cliffdude** [comment]
I had a similar issue, the sofar2mqtt would stop receiving data from the
inverter however I was still able to ping it over the wifi.
I changed the power adapter for the 5v and the problem stopped.

**Reply by jacoscar** [reply]
hi, is it still working for you?
